### PR TITLE
retain the logs for more builds 

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Add more build to be retained in the jenkins logs

per @agnivade request

### CODE REVIEW

File: build/Jenkinsfile.branch [hunk 1]
```diff
@@ -18,7 +18,7 @@ pipeline {
   }
 
   options {
-    buildDiscarder(logRotator(numToKeepStr: '3'))
+    buildDiscarder(logRotator(numToKeepStr: '30'))
```
Comment (hanzei) [inline build/Jenkinsfile.branch]: What is the reasoning behind only keeping a certain number of build vs. keeping them for a certain number of day?

Comment (cpanato) [reply build/Jenkinsfile.branch]: no reason, maybe to keep easier the disk space on jenkins :)

but we can change for daysToKeepStr and put lets set 10 days

Comment (agnivade) [reply build/Jenkinsfile.branch]: Yes, let's do that instead and set the number to 15. We can decrease it if we see there is pressure on the disk space.

**Pull request merged**